source/agora/test/Base.d

```diff
@@ -1109,34 +1109,35 @@ public class TestAPIManager
 
         Params:
             client_idxs = client indices for the nodes to be checked
-            height = expected block height of the nodes
+            to_height = expected block height of the nodes
```

Comment:
It's a lot more natural to do `from .. to` just like with the array syntax.

Reply:
I agree and was tempted to add `from_height` before `to_height` in the parameters but was trying to avoid  an extra overload as default is wanted for 'from_height' in most cases.

Reply:
Ah I see.

Reply:
Looking at the code I think that `from` is last so it can have a default value.

Another point, try not to repeat the type in the name, e.g. `Height to, Height from` is clear enough... I think ?

Reply:
Updated names.